sys/auto_init/Makefile

```diff
@@ -22,4 +22,8 @@ ifneq (,$(filter auto_init_usbus,$(USEMODULE)))
   DIRS += usb
 endif
 
+ifneq (,$(filter auto_init_security,$(USEMODULE)))
+  DIRS += security
```

Comment:
I don't understand the need for this new auto_init_security.

Reply:
Among others, we do also have separate `auto_init_can` or `auto_init_usb` so I don't see a problem with that. Further, we might add other security related initialization in here later. To be more precise, I personally have some ongoing work which might go in there.